domain/src/main/java/org/woowatechcamp/banchan/domain/entity/DishInfo.kt

```diff
@@ -0,0 +1,11 @@
+package org.woowatechcamp.banchan.domain.entity
+
+data class DishInfo(
+    val id: String,
+    val name: String,
+    val thumbnailUrl: String,
+    val description: String,
+    val salesPercent: String,
+    val originalPrice: String,
+    val salesPrice: String,
+)
```

Comment:
```suggestion
data class DishInfo(
    val id: String,
    val name: String,
    val thumbnailUrl: String,
    val description: String,
    val originalPrice: Int?,
    val salesPrice: Int,
) {
    val salesPercent: Int
        get() = if (originalPrice != null) {
            (((originalPrice - salesPrice).toDouble() / originalPrice) * 100).roundToInt()
        } else {
            0
        }
}
```

`salesPercent`를 `DishInfo`의 생성자에 넣지 말고 멤버 변수로 선언 한 뒤 `get()`으로 계산해서 참조하는 것은 어떨까요?

Reply:
계산로직은 확실히 좋은 것 같아요! 그런데 Price의 경우 String 생성자를 갖게 만들 것 같습니다~ 그래야 뷰에서 사용하기 편리할 것 같아요!